spec/basic_examples_spec.rb

```diff
@@ -0,0 +1,47 @@
+require "hypothesis"
+
+def expect_failure(&block)
+  expect(&block).to raise_exception(RSpec::Expectations::ExpectationNotMetError)
+end
+
+RSpec.configure do |c|
+  c.include(Hypothesis)
+  c.include(Hypothesis::Providers)
+end
+
+RSpec.describe "some basic hypothesis tests" do
+  it "should think integer addition is commutative" do
+    hypothesis do 
+      x = given integers
```

Comment:
I'd say it's more Rubyish to pass `integers` as a message: `given :integers` instead of `given integers`. For passing parameters, you'd either do it with a nested hash, `given(integers: {min: 4})` or FactoryBot style: `given(:integers, min: 5)`. I think the second would be more appropriate.

Reply:
I am a very strong believer in just having functions which return values and where you can manipulate them as normal values. I don't think I'm likely to change that regardless of the local idiom.

Reply:
I also agree that integers should be a function, or an object.